modules/ROOT/pages/deployment/index.adoc

```diff
@@ -107,6 +107,10 @@ If you need to restart oCIS because of configuration changes in `/etc/ocis/ocis.
 
 The logs of oCIS can be displayed by issuing `journalctl -f -u ocis`.
 
+include::{latest-server-version}@server:admin_manual:page$installation/deployment_recommendations/nfs.adoc[leveloffset=+2]
```

Comment:
```suggestion
include::{latest-server-version}@server:admin_manual:installation/deployment_recommendations/nfs.adoc[leveloffset=+2]
```
`page$` is not mandatroy needed (exception) while other ressource identifyers like `example$` or `partial$` are

Reply:
I have learned, that if there is no family resource ID present, it uses a reference to the pages resource automatically